packages/@uppy/companion/package.json

```diff
@@ -59,7 +59,7 @@
     "ms": "2.1.3",
     "node-schedule": "2.1.1",
     "prom-client": "15.1.2",
-    "serialize-error": "^11.0.0",
+    "serialize-error": "^2.1.0",
```

Comment:
Do we need to downgrade 9 major versions?

Reply:
it was upgraded here 928eff7c58125635560006f71c8d8306ecdc5c63 however that `await import` caused jest to crash